node/resolvers/items.ts

```diff
@@ -253,6 +253,24 @@ export const mutations = {
     )
 
     return data
+  },
+  setManualPrice: async (
+    _: unknown,
+    { input: {itemIndex, price} }: { input: {itemIndex: number, price: number} },
```

Comment:
can you check if eslint is properly configured in your computer? prettier should've added some spaces around these curly brackets

Reply:
yep, you right!